src/Curriculum/Module/Module.tsx

```diff
@@ -63,13 +64,24 @@ const Module: FC<ModuleIntf & Props> = (
   },
 ) => {
   const [isOpen, setOpen] = useState(false);
+  const [isOpenStored, setOpenStored] =
+    useSessionStorage<boolean>(`module-${id}-${index}`, false,
+      value => JSON.parse(value));
+
+  useEffect(() => {
+    setOpen(isOpenStored);
+  }, [isOpenStored]);
```

Comment:
I think that you can get rid of the standard `useState` variable `isOpen` and just use the `useSessionStorage` hook and that would get rid of the `useEffect` to update `isOpen`

```suggestion
  const [isOpen, setOpenStored] =
    useSessionStorage<boolean>(`module-${id}-${index}`, false,
      value => JSON.parse(value));
```